docs/01-introduction.Rmd

```diff
@@ -80,6 +80,81 @@ You have to know three most basic concepts for a Hugo-based website:
 If you are satisfied with this default theme, you are basically ready to start writing and publishing your new website! We will show how to use other themes in Section \@ref(other-themes). However, please keep in mind that a more complicated and fancier theme may require you to learn more about all the underlying technologies like the Hugo templating language, HTML, CSS, and JavaScript.
 
+## A quick deploy
+
+Publishing a **blogdown** website involves copying the output directory containing all your rendered site files (typically, a bunch of `*.html` files and site assets like fonts, CSS, JavaScript, and images) to a web server or web hosting service.
+
+In this section, we'll walk you through a quick deploy to publish your website using a drag-and-drop tool from \index{Netlify} https://www.netlify.com. Netlify is a platform that offers cloud hosting and serverless backend services for static websites. Lucky for us, that is exactly what we need- since **blogdown** helps us create static websites using Hugo. 
+
+If you followed along with Chapter \@ref(a-quick-example), you should now have:
+
+1. A static website...
+1. Using the Hugo Lithium theme...
+1. With knitted `.Rmd` content for one post...
+1. Served locally using **blogdown**. 
+
+The drag-and-drop method we describe next is a good workflow to start with if you are not familiar with GIT or GitHub, and if you are not familiar with deploying websites. There are many options for publishing static websites, and we will talk more about them plus advanced workflows involving version control and continuous deployment in Chapter \@ref(deployment).
+
+### Build site
+
+The publishing directory of a Hugo site is `public/` by default. Your *built* website will be generated to this directory, meaning that you do not need to manually add any files to this directory. You can upload everything under `public/` to any web server that can serve static websites, and your website will be up and running. 
+
+Serving your Hugo site locally using `blogdown::serve_site()` does not generate the `public/` directory. To follow this drag-and-drop method, you'll need to build your site locally first.
+
+Restart the R session, and run:
+
+```{r eval=FALSE}
+blogdown::build_site()
+```
+
+Alternatively, if you use RStudio, you may use:
+
++ the "Build Website" button in the Build pane, or 
+
++ press `Ctrl + Shift + B` or `Cmd + Shift + B` to build the website.
+
+You should see a new `public/` directory under the root directory of your project. Never edit files in this directory, as your changes will always be overwritten whenever you re-build your site.
+
+Note that this function does not rerender R Markdown files by default, because it may be expensive and often undesirable to render `.Rmd` files that have been rendered before. Given that R package upgrades have a tendency to break older code, continuously re-rendering old content is nearly impossible to do without errors, especially over longer periods of time. 
```

Comment:
I think here we should link to the part where we will talk about workflow, based on content in https://blog.rstudio.com/2021/01/18/blogdown-v1.0/#workflows

I think that because as reader I ask myself "and what if I want to rerender the Rmd ?" - this is a getting started section, go learn more there. 

I know the part is written yet, but I think it could be interesting to maybe insert comment or dummy link for now so that we start cross linking those. 

What do you think ?

Reply:
I'll clarify in this text - I don't think all of that link is helpful at this point.

Reply:
Ok not need to add anything then. I just wanted to share the thought.